src/js/iqons.js

```diff
@@ -205,7 +205,7 @@ ${ content }
         if (!!String.prototype.padEnd) return string.padEnd(maxLength, fillString);
         else {
             while (string.length < maxLength) string += fillString;
-            return string;
+            return string.substr(0, maxLength);
```

Comment:
That's not how the native `padEnd` behaves.
Also `substr` is a legacy function. Use `substring` instead.